src/api/hydration.ts

```diff
@@ -8,17 +8,34 @@ import {
 } from 'services/supabase';
 import { ENTITY } from 'types';
 
-type ConnectorTagData = Pick<
+// TODO (naming): Consider removing he tight coupling between this file and the stores.
+//  These APIs are truly general purpose. Perhaps break them out by supabase table.
+type ConnectorTagResourceData = Pick<
```

Comment:
I left behind this TODO comment because, shortly after pushing the last hydration-related commit, I felt the codebase would be better served by moving these functions into a file whose contents are governed by a specific supabase table. That is the standard today for the _src/api_ directory. Since these functions are commonplace, table-specific APIs, they framed in that fashion.